documentation/docs/libraries/nodejs/exchange_guide.md

```diff
@@ -0,0 +1,243 @@
+---
+description: Easily integrate with your exchange, custody solution, or product using the wallet.rs library.
+keywords:
+- integrate
+- exchange
+- account model
+- addresses
+- wallet.rs
+- setup
+- NodeJS
+- explanation
+---
+import CodeBlock from '@theme/CodeBlock';
+import generate_mnemonic from  '!!raw-loader!./../../../../bindings/nodejs/examples/exchange/1-generate-mnemonic.js';
+import create_account from  '!!raw-loader!./../../../../bindings/nodejs/examples/exchange/2-create-account.js';
+import generate_address from  '!!raw-loader!./../../../../bindings/nodejs/examples/exchange/3-generate-address.js';
+import check_balance from  '!!raw-loader!./../../../../bindings/nodejs/examples/exchange/4-check-balance.js';
+import listen_events from  '!!raw-loader!./../../../../bindings/nodejs/examples/exchange/5-listen-events.js';
+import send_amount from  '!!raw-loader!./../../../../bindings/nodejs/examples/exchange/6-send-amount.js';
+
+# Exchange Guide
+
+## The IOTA Wallet Library
```

Comment:
Does it make sense to clarify everywhere that this is also for Shimmer? Might be confusing for Shimmer integrators that we only talk about IOTA.

Reply:
Sure will be changed